rest-api/build.gradle.kts

```diff
@@ -2,19 +2,15 @@ plugins {
     id("org.jetbrains.kotlin.jvm")
     id("org.jetbrains.kotlin.kapt")
     id("com.github.johnrengelman.shadow") version "7.1.2"
-    id("io.micronaut.application") version "3.4.1"
+    id("io.micronaut.application") version "3.6.3"
 
 }
 
-version = "0.1"
-group = "org.modelix.samples.some"
-json_bulk_access_version = project.properties["json_bulk_access_version"]
+//group = "org.modelix.samples.some"
```

Comment:
Why commented?

Reply:
I don't thin it is actually needed for anything?

Reply:
the `rest-api` will be reworked in the future anyways